chameleonultragui/ios/Podfile

```diff
@@ -0,0 +1,44 @@
+platform :ios, '15.0'
```

Comment:
Does this mean that we have minimal iOS version 15? Or it is just target version?

Reply:
This was the default. If I go older XCode complains when it builds, maybe some libraries are not compatible, I didn't look in detail. We are on iOS 16+ now which supports the majority of devices in use.